### PR TITLE
Add document explaining how to help

### LINKED ISSUE #52: Create a "How can I contribute" guide
Create a file  (and also link it in our homepage) where we can redirect people, who want to help

### PR DESCRIPTION
**Pull request by DDCreationStudios**
close #52

### CODE REVIEW

File: docs/Helping.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# How to help
+
+This document is a starting point for any non-profit contributions to this organization.
+
+## What do you want to do? 
+
+First of thank you very much for decided to contribute to society and make the world a better place.
```
Comment (robaxelsen) [inline docs/Helping.md]: Small correction: change "decided" to "having decided" or "deciding".

File: docs/Helping.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Now you have to ask yourself what you enjoy doing and what your skillset is. 
+
+Here are some examples on what you can do:
```
Comment (robaxelsen) [inline docs/Helping.md]: Correction: Change "on" to "of".

File: docs/Helping.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+- Writing 
+  - You can improve existing written documents or create new ones. We have a medium publication you can also use for writing articles about whatever you like
+- Video/Photo/Audio
+  - You can either create content of our meetups or edit existing ones. 
+- Organizing /Event-managing
+  - You can use your skills to help us coordinate all events. 
+  - Those tasks cover emails, meetings, phone calls, skype calls 
+- Project Development
+  - Start your own project and be the project manager of it. 
+  - Present you project and get feedback
```
Comment (robaxelsen) [inline docs/Helping.md]: Correction: change "you" to "your"

File: docs/Helping.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+- Innovator
+  - help with new ideas we can work on
+  - create detailed plans on how to accomplish them
+- Everything else
+  - You are skilled in some area? Simply show us 
+
+## Show us
```
Comment (robaxelsen) [inline docs/Helping.md]: I would personally rewrite this or remove the section entirely. I understand your motivation for including it, but I think it potentially can deter more people than motivate. Plus I don't think we should put limitations/requirements on people for them to help. The fact that they are wanting to contribute will hopefully lead them to this document, and here we are already listing ways for them to contribute.

File: docs/Helping.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+We meet 1000 of people, who only talk. Show us related projects or your motivation for the desired task. Come up with detailed plans on how to accomplish the task. 
+
+## Don't be afraid to fail
+
+You can only win to try new things. Just start and feel the excitement of accomplishing something great in life.
```
Comment (robaxelsen) [inline docs/Helping.md]: Correction: change "win to try" to "win by trying".

**Comment by robaxelsen** [comment]
@DDCreationStudios See my review comments above. Overall great work on the document, and thanks for the initative :tada:

File: docs/Helping.md (hunk 1 above)
Comment (DDCreationStudios) [reply docs/Helping.md]: done

File: docs/Helping.md (hunk 2 above)
Comment (DDCreationStudios) [reply docs/Helping.md]: done

File: docs/Helping.md (hunk 3 above)
Comment (DDCreationStudios) [reply docs/Helping.md]: done

File: docs/Helping.md (hunk 5 above)
Comment (DDCreationStudios) [reply docs/Helping.md]: done

File: docs/Helping.md (hunk 4 above)
Comment (DDCreationStudios) [reply docs/Helping.md]: agree - removed it

**Comment by DDCreationStudios** [comment]
@robaxelsen All comments are resolved. Please check and merge if you are ok with it :) 🚀

**Pull request merged**